cmd/buildctl/common/common.go

```diff
@@ -88,3 +92,25 @@ func ResolveClient(c *cli.Context) (*client.Client, error) {
 
 	return client.New(ctx, c.GlobalString("addr"), opts...)
 }
+
+func ParseTemplate(format string) (*template.Template, error) {
+	// aliases is from https://github.com/containerd/nerdctl/blob/v0.17.1/cmd/nerdctl/fmtutil.go#L116-L126 (Apache License 2.0)
+	aliases := map[string]string{
```

Comment:
Do we need this? I think the examples in the PR work without this as well.

Reply:
This is not new in this PR, just moved from `cmd/buildctl/debug/workers.go`
https://github.com/moby/buildkit/pull/2992/commits/405565c005a508675f3725fd2eef467dbbfbf02c